js/apps/admin-ui/maven-resources/theme-resources/messages/messages_ca.properties

```diff
@@ -0,0 +1,275 @@
+#converted from translation.json
+#Thu Oct 26 10:13:57 CEST 2023
```

Comment:
I think we can remove these headers no? I don't think they'll provide much value for future reference.